src/aat/resources/features/F-1015 - Start Case Event TTL/F-1015.feature

```diff
@@ -17,6 +17,20 @@ Feature: F-1015: Update Case - Start Case Event - Update Code for TTL
     And   the response has all other details as expected
     And   the response [contains the TTL.SystemTTL for the case, that has been set to 20 days from today]
 
+  @S-1015.1.repeat 
+  Scenario: TTLIncrement is set to "20" for the Case Event and Start Event is invoked on v1_external#/case-details-endpoint/startEventForCaseworkerUsingGET
+    Given a user with [an active profile in CCD],
+    And   a successful call [to create a case] as in [F-1015_CreateCasePreRequisiteCaseworker_noTTL]
+    When  a request is prepared with appropriate values,
+    And   the request [contains correctly configured event details]
+    And   the request [has a TTLIncrement of 20 days configured]
+    And   the request [is configured to trigger an About To Start callback that does not change of the TTL.suspended or TTL.OverrideTTL or TTL.SystemTTL]
```

Comment:
that does not change *any*

Reply:
Resolved in 5af480c67d8ab643d8553b812272b8f7ec1932a4